package.json

```diff
@@ -75,11 +76,11 @@
         }
     },
     "devDependencies": {
-        "@gouvfr/dsfr-chart": "^1.0.0",
         "@babel/core": "^7.20.2",
         "@emotion/react": "^11.10.4",
         "@emotion/styled": "^11.10.4",
         "@gouvfr/dsfr": "1.13.2",
+        "@gouvfr/dsfr-chart": "^1.0.0",
```

Comment:
Please, do not generate unessesary diff, I know it must be your package manager reordering but we want to be able to trace the version changes.